app/src/main/res/values/strings.xml

```diff
@@ -413,7 +413,7 @@
   <!--- Sync Indicator-->
   <string name="syncindicator_status_pending">Sync pending</string>
   <string name="syncindicator_status_failed">Sync failed</string>
-  <string name="syncindicator_status_syncing">Syncing…</string>
```

Comment:
The design still has three dots when syncing, did we remove this by mistake?

Reply:
These are the latest designs: https://www.figma.com/file/vmZlmlFSCOrEYzTMkqsNB8/Simple-Production?node-id=18287%3A32415